client/my-sites/stats/activity-log-item/style.scss

```diff
@@ -119,7 +119,7 @@
 }
 
 .activity-log-item__actor-role {
-	text-transform: uppercase;
+	text-transform: capitalize;
```

Comment:
Roles often come in lowercase, such as `administrator` or `editor` so this is basically a way of making those consistent.